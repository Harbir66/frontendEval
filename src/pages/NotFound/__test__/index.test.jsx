import React from 'react';
import { render } from '@testing-library/react';
import PageNotFound from '../index';

describe('Error Page', () => {
  it('should render', () => {
    const { asFragment } = render(<PageNotFound />);
    expect(asFragment()).toMatchSnapshot();
  });
});
