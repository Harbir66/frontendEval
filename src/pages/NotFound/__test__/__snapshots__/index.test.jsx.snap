// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Page should render 1`] = `
<DocumentFragment>
  <div
    class="main"
  >
    <header
      class="padding"
    >
      EVENTIFY
    </header>
    <div
      class="pageNotFoundContainer"
    >
      <p>
        404 Error. Page not found
      </p>
    </div>
    <footer
      class="padding"
    >
      <p>
        THEMES
      </p>
      <div
        class="themes"
      />
    </footer>
  </div>
</DocumentFragment>
`;
